mantidproject/mantid
# Dashboard publish ctest log

**Description of work.**
Added function in monitoring mantid test

**Report to:**  thomashampson

**To test:**

build the branch on Jankins pipeline, the 3 files ["darwin17.log", "linux-gnu.log", "msys.log"] should appear in
Build Artifacts in the build with ctest log from the the 3 machine

If testing on the `build_packages_from_branch` pipeline, you can see if the dashboard updates at 14:00 UTC on https://donaldchung-hk.github.io/mantid-monitor-dashboard/index.html

Partly Fixes #33793.  The other half of the solution is the dashboard and storage system in https://github.com/DonaldChung-HK/mantid-monitor-dashboard.

*This does not require release notes* because this is an internal change to the Jenkins build pipeline

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
